CHANGELOG.md

```diff
@@ -56,52 +58,54 @@ Addressed Feedback: Vignette ([`779314a`](https://github.com/UBC-MDS/autopredict
+* fix: Feedback addressed by Zeily through updating the description of how the Best Score function works in vingette ([`dca42f2`](https://github.com/UBC-MDS/autopredictor/commit/dca42f20fc58611e3141431e441bfb3b6b8264f6))
 
-* fix: Feedback addressed by adding the references and credits onto the Vignette ([`71a16c4`](https://github.com/UBC-MDS/autopredictor/commit/71a16c4d7cc641d371d453286aab6ace57863c51))
+* fix: Feedback addressed by Zeily through adding the references and credits onto the Vignette ([`71a16c4`](https://github.com/UBC-MDS/autopredictor/commit/71a16c4d7cc641d371d453286aab6ace57863c51))
 
-* fix: added reference for the logistic of each functions ([`73ef1d5`](https://github.com/UBC-MDS/autopredictor/commit/73ef1d57124a3e55af3c65e4dc7d5aabc7b9087d))
+* fix: Feedback addressed by Sharon through adding reference for the logistic of each functions in README ([`73ef1d5`](https://github.com/UBC-MDS/autopredictor/commit/73ef1d57124a3e55af3c65e4dc7d5aabc7b9087d))
 
-* fix: Added references to README ([`b07a6ce`](https://github.com/UBC-MDS/autopredictor/commit/b07a6ce4525dbbba68655dd0191cf1d0c93b26ef))
+* fix: Feedback addressed by Sharon through adding references to README ([`b07a6ce`](https://github.com/UBC-MDS/autopredictor/commit/b07a6ce4525dbbba68655dd0191cf1d0c93b26ef))
 
-* fix: Added dependencies list to README ([`220709e`](https://github.com/UBC-MDS/autopredictor/commit/220709ef9a088bb11b3fcae2bf9b413d0dffb0be))
+* fix: Feedback addressed by Sharon through adding dependencies list to README ([`220709e`](https://github.com/UBC-MDS/autopredictor/commit/220709ef9a088bb11b3fcae2bf9b413d0dffb0be))
 
-* fix: includes instruction to run unit test and updated installation instruction through PyPI ([`0f192be`](https://github.com/UBC-MDS/autopredictor/commit/0f192bec6912a8182f260926409432e90ffce3d4))
+* fix: Feedback addressed by Sharon through including instruction to run unit test and updated installation instruction through PyPI in README ([`0f192be`](https://github.com/UBC-MDS/autopredictor/commit/0f192bec6912a8182f260926409432e90ffce3d4))
 
-* fix: includes reference to vignette for more detailed explanation for each model ([`871cf14`](https://github.com/UBC-MDS/autopredictor/commit/871cf14c550db2a30971e897518fd1f70fe68de5))
+* fix: Feedback addressed by Sharon through including reference to vignette for more detailed explanation for each model ([`871cf14`](https://github.com/UBC-MDS/autopredictor/commit/871cf14c550db2a30971e897518fd1f70fe68de5))
 
-* fix: included the list of regression models and scoring metrics used and specified response type in summary ([`b29e608`](https://github.com/UBC-MDS/autopredictor/commit/b29e6083e27ffab70226f15eb39acdeb01ea07e0))
+* fix: Feedback addressed by Sharon through including the list of regression models and scoring metrics used and specified response type in summary of README ([`b29e608`](https://github.com/UBC-MDS/autopredictor/commit/b29e6083e27ffab70226f15eb39acdeb01ea07e0))
 
 ### Unknown
 
```

Comment:
Anu's commits under the unknown section are fix commits and thus should be under the Fix section

Reply:
I couldnt add the suggestion but Anu's fix commits should have spaces between 'fix:' and 'Feedback'

* Merge pull request #74 from UBC-MDS/about-readme
fix: Feedback addressed by Arturo for README About section ([`26dd68c`](https://github.com/UBC-MDS/autopredictor/commit/26dd68c258f088c785af08b295eb56da950178da))
* fix: Feedback addressed by Anu. Small modification in docstring ([`a474230`](https://github.com/UBC-MDS/autopredictor/commit/a4742302f7e1dd22033ecdce59a412c6bb9497ce))
* fix: Feedback addressed by Anu. creatd new test for R2 scoring metric ([`086cd5b`](https://github.com/UBC-MDS/autopredictor/commit/086cd5b5d9e82bd207e4e54e98ca22ae9940cc08))
* fix: Feedback addressed by Anu. Made changes as per review suggestion ([`ab4dc34`](https://github.com/UBC-MDS/autopredictor/commit/ab4dc340ed4c7ae5272bc2bdd5dd3bad68c0eda5))